src/context/isUserAuthorized.tsx

```diff
@@ -29,15 +29,15 @@ const IsUserAuthorizedContext: FC<Props> = ({ children }) => {
     };
     fetchData();
 
-    return (() => {
+    return () => {
       setIsUserAuthorized(false);
-    });
+    };
   }, []);
-  return(
+  return (
     <isUserAuthorizedContext.Provider value={isUserAuthorized}>
       {children}
     </isUserAuthorizedContext.Provider>
-  )
-}
+  );
+};
```

Comment:
why is this file changed?

Reply:
I didn't change  the file but I might have run lint it must have changed it